packages/cli/src/templates/cmsNameMapper.ejs

```diff
@@ -0,0 +1,31 @@
+// const componentsMap = <%- JSON.stringify(props.componentsMap) %>
+const componentsMap = {
```

Comment:
what's .ejs format? :)

Reply:
it's used for templates https://ejs.co/
this template generates us dynamically proper components resolving :)